src/edu/csus/ecs/pc2/clics/API202306/CLICSOrganization.java

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.io.File;
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.model.Account;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Organization
+ * Contains information about an Organization (institution)
+ *
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSOrganization {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String icpc_id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String formal_name;
+
+    @JsonProperty
+    private String country;
+
+    @JsonProperty
+    private CLICSFileReference [] country_flag;
+
+    @JsonProperty
+    private String url;
+
+    @JsonProperty
+    private String twitter_hashtag;
+
+    @JsonProperty
+    private String twitter_account;
+
+    @JsonProperty
+    private CLICSLocation location;
+
+    @JsonProperty
+    private CLICSFileReference [] logo;
+
+    public CLICSOrganization() {
+        // for Jackson deserializer
+    }
+
+    /**
+     * Fill in organization information properties
+     * TODO: fix to use organization obtained from organizations.json.  From insitutions.tsv, we get this:
+     * [0]:1329 [1]:New York University [2]:NYU
+     * but only need [1].  We get the rest from the a typical account.
```

Comment:
Not sure what `...from the a typical account` is trying to say.  And is it talking about a PC2 account?

Reply:
Clarified the comment in next push.